examples/data-sources/stackit_security_group/resource.tf

```diff
@@ -0,0 +1,4 @@
+resource "stackit_security_group" "example" {
```

Comment:
this file should be called `datasource.tf`

Reply:
renamed the file